generators/heroku/index.js

```diff
@@ -31,31 +31,49 @@ module.exports = HerokuGenerator.extend({
         this.angularAppName = this.getAngularAppName();
         this.buildTool = this.config.get('buildTool');
         this.applicationType = this.config.get('applicationType');
+        this.herokuAppName = this.config.get('herokuAppName');
     },
 
     prompting: function () {
         var done = this.async();
 
-        var prompts = [
-            {
-                type: 'input',
-                name: 'herokuDeployedName',
-                message: 'Name to deploy as:',
-                default: this.baseName
-            },
-            {
-                type: 'list',
-                name: 'herokuRegion',
-                message: 'On which region do you want to deploy ?',
-                choices: ['us', 'eu'],
-                default: 0
-            }];
-
-        this.prompt(prompts).then(function (props) {
-            this.herokuDeployedName = _.kebabCase(props.herokuDeployedName);
-            this.herokuRegion = props.herokuRegion;
-            done();
-        }.bind(this));
+        if (this.herokuAppName && this.herokuAppName.length != 0) {
```

Comment:
Length check is not required as `if (this.herokuAppName)` itself checks for undefined, null and empty